modules/nodes-pods-secrets-updating.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-secrets.adoc
+
+[id='nodes-pods-secrets-updating_{context}']
+=  Updating secrets
```

Comment:
Is this module supposed to be a procedure? It either needs steps or a new title.